beat-editor/src/components/BeatChart/BeatChart.jsx

```diff
@@ -367,6 +397,80 @@ const BeatChart = ({
             deleteModeCoordinates={deleteModeCoordinates}
             unusableSegments={unusableSegments}
           />
+        
+        {/* keyboard shortcuts */}
```

Comment:
I saw what you meant about can possibly make into a reusable component. Can we put this in it's own component and call it here instead? That way it'll ease up the clutter and we can revisit breaking down the separate component file and make it into a reusable component